src/main/java/racingcar/policy/DefaultRandomValueMovePolicy.java

```diff
@@ -0,0 +1,20 @@
+package racingcar.policy;
+
+import racingcar.random.RandomValueGenerator;
+
+public class DefaultRandomValueMovePolicy implements RandomValueMovePolicy {
```

Comment:
```suggestion
public class RandomValueMovePolicy implements MovePolicy {
```
인터페이스에 굳이 RandomValue를 드러내지 않아도 되지 않을까?